Failure in test-python-utils with v2.6.0a2

<!-- Please fill in the following information to report a problem with Cantera. If you have a question about using Cantera, please post it on our Google Users' Group (https://groups.google.com/forum/#!forum/cantera-users). Feature enhancements should be discussed in the dedicated Cantera enhancements repository (https://github.com/Cantera/enhancements/new/choose) -->

**Problem description**
Two tests fail on my mac  (there are others that fail, but they are comments in the source code that these are expected such as in `test_mixture.py::TestMixture::test_equilibrate2`)

<!-- A clear and concise description of what the bug is. -->

**Steps to reproduce**

<!-- A minimal, complete, and reproducible example demonstrating the problem. See https://stackoverflow.com/help/minimal-reproducible-example for additional suggestions on how to create such an example. -->

run `scons test test-python-utils`

**Behavior**

See the screen dump at the end of this message for more info
<!-- Describe the result of executing the above steps, and how this differs from what you expect to happen. -->

**System information**

- Cantera version: 2.6.0a2 commit #627e21e95
- OS: Mac OS 10.15.7
- Python/MATLAB/other software versions: 3.7.4

**Attachments**

<!-- If applicable, attach scripts and/or input files to help explain your problem. Please do *not* attach screenshots of code or terminal output. -->

```
../../build/python/cantera/test/test_transport.py .......................................                        [ 96%]
../../build/python/cantera/test/test_utils.py ..................FF.......                                        [100%]

======================================================= FAILURES =======================================================
______________________________________________ TestPyToAnyValue.test_none ______________________________________________

self = <cantera.test.test_utils.TestPyToAnyValue testMethod=test_none>

    def test_none(self):
>       self.check_conversion(None, "void")

../../build/python/cantera/test/test_utils.py:30: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../build/python/cantera/test/test_utils.py:14: in check_conversion
    self.assertEqual(held_type, check_type)
E   AssertionError: 'v' != 'void'
E   - v
E   + void
__________________________________________ TestPyToAnyValue.test_scalar_bool ___________________________________________

self = <cantera.test.test_utils.TestPyToAnyValue testMethod=test_scalar_bool>

    def test_scalar_bool(self):
>       self.check_conversion(True, "bool")

../../build/python/cantera/test/test_utils.py:55: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../build/python/cantera/test/test_utils.py:14: in check_conversion
    self.assertEqual(held_type, check_type)
E   AssertionError: 'b' != 'bool'
E   - b
E   + bool
```

**Additional context**

<!-- Add any other context about the problem here. -->

--- a/SConstruct
+++ b/SConstruct
@@ -1052,7 +1052,7 @@ if not env['BOOST_LIB_VERSION']:
                  " 'boost_inc_dir' to point to the boost headers.")
 else:
     print('INFO: Found Boost version {0}'.format(env['BOOST_LIB_VERSION']))
-# demangle is availble in Boost 1.55 or newer
+# demangle is available in Boost 1.56 or newer
 env['has_demangle'] = conf.CheckDeclaration("boost::core::demangle",
                                 '#include <boost/core/demangle.hpp>', 'C++')
 

--- a/src/base/AnyMap.cpp
+++ b/src/base/AnyMap.cpp
@@ -488,8 +488,10 @@ struct convert<Cantera::AnyValue> {
 namespace Cantera {
 
 std::map<std::string, std::string> AnyValue::s_typenames = {
+    {typeid(void).name(), "void"},
     {typeid(double).name(), "double"},
     {typeid(long int).name(), "long int"},
+    {typeid(bool).name(), "bool"},
     {typeid(std::string).name(), "string"},
     {typeid(vector<AnyValue>).name(), "vector<AnyValue>"},
     {typeid(vector<AnyMap>).name(), "vector<AnyMap>"},

